api/api/specs/ensemblers.yaml

```diff
@@ -32,6 +32,10 @@ paths:
           schema:
             type: integer
             default: 10
+        - in: query
+          name: search
+          schema:
+            type: string
```

Comment:
Added this to the API endpoint so the ensembler search bar should now work.

Reply:
Would be good to include a description for this field as it's not immediately clear from the name what we are searching. For example, something like:`Searches ensembler name for a partial match of the search text`